addons/web/static/tests/views/graph_view_tests.js

```diff
@@ -3478,4 +3480,41 @@ QUnit.module("Views", (hooks) => {
         ]);
         checkDatasets(assert, graph, "data", { data: [56, 26, 4, 105, 48] });
     });
+
+    QUnit.test("fill_temporal is true by default", async function (assert) {
+        assert.expect(1);
+
+        const graph = await makeView({
+            serverData,
+            type: "graph",
+            resModel: "foo",
+        });
+
+        const { context } = getGraphModelMetaData(graph);
+        assert.strictEqual(
+            context.fill_temporal,
+            true,
+            "By default, all the empty temporal slices should be visible"
+        );
+        // Can't test the fixture cause the fill_temporal feature is not in the mock server.
+    });
```

Comment:
I prefer a test that checks the *observable* state instead of the internal. Here, the observable state that matters is the context given to the web_read_group rpc. Actually, it is already tested in two tests above, but I don't mind having another specific test for this. Of course, no need to check the resulting graph as we don't care whether or not the MockServer correctly handles that feature (which it doesn't, as you stated).